backend/test/conftest.py

```diff
@@ -23,6 +24,10 @@
 from ml_space_lambda.enums import DatasetType
 
 
+def pytest_generate_tests(metafunc):
+    os.environ["LAMBDA_TASK_ROOT"] = "./src/"
```

Comment:
Can you add a comment about why this is needed?

Reply:
That's fine for and probably makes sense for things like the region etc. I was more interested in why we had to set the `LAMBDA_TASK_ROUTE` as that's new with this PR.

Reply:
It was needed because the Sagemaker SDK check invoked with `retrieve` checks file names and it was unable to get the env needed for locating the file names

Reply:
Can you add the comment in code so if someone sees it they'll know why it's there. Unlike some other env variables it's not totally clear why you'd need this.